go/vt/vtgate/grpcvtgateservice/server.go

```diff
@@ -110,12 +110,6 @@ func (vtg *VTGate) Execute(ctx context.Context, request *vtgatepb.ExecuteRequest
 	if session == nil {
 		session = &vtgatepb.Session{Autocommit: true}
 	}
-	if session.TargetString == "" && request.TabletType != topodatapb.TabletType_UNKNOWN {
-		session.TargetString = request.KeyspaceShard + "@" + topoproto.TabletTypeLString(request.TabletType)
-	}
-	if session.Options == nil {
-		session.Options = request.Options
-	}
```

Comment:
@vitessio/query-serving 
Do we have tests for the grpc way of executing queries? Note that this change removes the "option" of providing options in the grpc request instead of in the session.

Reply:
I was able to find some in `go/vt/vitessdriver` and `go/test/endtoend/vtgate/godriver/main_test.go` There might be more, but all will be old tests only.

Reply:
good enough, I guess. they tell us I didn't break the grpc way of executing queries.